src/Data/Integer/Divisibility.agda

```diff
@@ -17,11 +17,17 @@ open import Relation.Binary.PropositionalEquality
 
 -- Divisibility.
 
-infix 4 _∣_
+infix 4 _∣_ _∣′_
```

Comment:
Hmm so I think in most places the library uses \prime rather than '?

Reply:
Yep. :( I tried to preserve backwards compatibility.

Alternatively we could put all the prime'd stuff into e.g. `Data.Integer.Divisibility.Signed` and call it `_∣_` too.

Reply:
That seems like a better option. I'll do that.